src/App.tsx

```diff
@@ -1,14 +1,73 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { getTodos, getUser } from './api';
+
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+type FullTodo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+  user: User | null;
+};
```

Comment:
you already have the todo type in [src/types/FullTodo.ts](https://github.com/mate-academy/react_dynamic-list-of-todos/pull/2285/files#diff-471dd23cc601795e9be4f7b4013fd6d19bd52a66da0491227ad7066620989ec6)

Reply:
```suggestion

```